manifests/03-rbac-role-cluster.yaml

```diff
@@ -84,6 +84,14 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
Based on the error this role needs to be added to the console-operator in its namespace.

```
Err getting olmconfig olmconfigs.operators.coreos.com "cluster" is forbidden: User "system:serviceaccount:openshift-console-operator:console-operator" cannot get resource "olmconfigs" in API group "operators.coreos.com" at the cluster scope
```